Add ubookmark to bookmarks/link sharing

telmich opened the pull request:
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x ] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``


- [x ] Your additions are ordered alphabetically.
- [ x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [ ] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [x ] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x ] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x ] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x ] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [ ] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [ ] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [ x] Any software project you are adding to the list is actively maintained.

tet3 commented:
The homepage and source links are the same; looks like the source link should be `https://code.ungleich.ch/ungleich-public/ubookmark/`

nodiscc commented on README.md:
```diff
@@ -235,6 +235,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
 - [Shaarli](https://github.com/shaarli/Shaarli) - Personal, minimalist, super-fast, no-database bookmarking and link sharing platform. ([Demo](https://demo.shaarli.org)) `Zlib` `PHP`
 - [Shiori](https://github.com/RadhiFadlillah/shiori) - Simple bookmark manager built with Go. `MIT` `Go`
 - [unmark](https://github.com/plainmade/unmark) - Open source to do app for links. `MIT` `PHP`
+- [ubookmark](https://ungleich.ch/u/projects/ubookmark/) - LDAP enabled bookmarking service. ([Demo (IPv6 only)](https://ipv6-blog.place7.ungleich.ch/), [Source Code](https://ungleich.ch/u/projects/ubookmark/)) `GPL-2` `Python`
```
Please remove the Demo (IPv6 only) link.
Please point the source code link to https://code.ungleich.ch/ungleich-public/ubookmark/
The correct license identifier is [GPL-2.0](https://spdx.org/licenses/GPL-2.0.html)

SuperSandro2000 replied:
Whats wrong with the demo link?

telmich commented:
* Removed demo link
* Updated source code link
* Updated license

I'll create a new PR, when we migrate https://IPv6.blog to use ubookmark in the next days. That will be reachable by IPv6 and also IPv4.

nodiscc commented:
Then I'd better wait, for everything to be reachable over V4/V6, then you can update this pull request, and I'll merge it.

That's an interesting case because technically IPv6-only is fine, however many people (percentage?) still have V4-only connections/setups (personally I always disable it because I won't bother setting up two IP stacks including firewalls etc. I know, it's bad habit).

From what I've seen mobile/4G connections are increasingly v6-only (or sometimes v4 NATed by the broadband provider), but do have a v6-to-v4 gateawy service provided by the ISP. hat led you to have this service available over IPv6 only?

SuperSandro2000 commented:
There is no IPv4 to IPv6.

I don't see the problem. At least where I live the majority has IPv6 and most tech savy people will also have it. None blocker for me.

nodiscc commented:
> I don't see the problem. At least where I live the majority has IPv6 and most tech savy people will also have it. None blocker for me.

```shell
$ ip -6 route 
fe80::/64 dev eth0 proto kernel metric 256  pref medium
$ dig -t AAAA ipv6-blog.place7.ungleich.ch
ipv6-blog.place7.ungleich.ch. 47 IN	CNAME	line.schottelius.org.
line.schottelius.org.	86389	IN	CNAME	line.vpn.schottelius.org.
line.vpn.schottelius.org. 85894	IN	AAAA	2a0a:e5c1:101::42
$ ip route get 2a0a:e5c1:101::42
RTNETLINK answers: Network is unreachable
```

I'm glad it works for you though

telmich commented:
ping. All parts fixed.
